pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -79,7 +79,7 @@ func (alloc *allocateAction) Execute(ssn *framework.Session) {
 func attemptToAllocateJob(ssn *framework.Session, stmt *framework.Statement, job *podgroup_info.PodGroupInfo) (allocated, pipelined bool) {
 	queue := ssn.Queues[job.Queue]
 
-	resReq := podgroup_info.GetTasksToAllocateInitResource(job, ssn.TaskOrderFn, true)
+	resReq := podgroup_info.GetTasksToAllocateInitResource(job, ssn.SubGroupOrderFn, ssn.TaskOrderFn, true)
```

Comment:
It's starting to look kinda weird that we inject those order functions to each podgroup.. These are global to the session and are the same for each podgroup. Maybe there's a better way? Inject it at object creation? Or store it globally? let's think

Reply:
We will get to it after subgroups are merged